.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: java
+
+notifications:
+  email: false
```

Comment:
In my experience, Travis CI spams you with build emails in a non-deterministic, untimely, and unhelpful fashion. I'm suggesting turning them off, at least to start.